Readme few changes (#2207)
CoMakery/comakery-server: README.md

- Yarn
- Chrome and [Chromedriver](https://chromedriver.chromium.org/)

> You can chech the last versions of yarn, node and else if needed in docker-compose.yml

Reviewer: ```suggestion
You can check actual versions of yarn, node and other tools in docker-compose.yml file
```

## Code Commit and Deployment Workflow

* Develop your code locally in a git feature branch
* Run `overcommit --install` and `overcommit --sign` to manage and configure Git hooks

> * Run quality checks and your tests (rspec, rubocop etc). The same checks CircleCI will run after you push your code to GitHub.

Reviewer: I think it's not necessary to run all tests locally it takes long time. Only that will fail for sure after your changes.
Other failed tests you can see in CI and fix it in next commit.

> * Go trough usual git flow to push your changes: `git add`, `git commit`, `git push` - to push changed to your branch 

Reviewer: ```suggestion
* Go through usual git flow to push your changes: `git add`, `git commit`, `git push` - to push changes to your branch 
```
